src/main/java/cloud/fogbow/ras/core/plugins/interoperability/azure/volume/AzureVolumePlugin.java

```diff
@@ -152,6 +177,17 @@ void updateInstanceAllocation(VolumeOrder volumeOrder) {
         }
     }
     
+    @VisibleForTesting
+    String defineResourceGroupName(Azure azure, String resourceName) {
+        String resourceGroupName = this.defaultResourceGroupName;
+        try {
+            resourceGroupName = AzureResourceGroupUtil.create(azure, this.defaultRegionName, resourceName);
+        } catch (Exception e) {
+            LOGGER.error(String.format(Messages.Error.ERROR_MESSAGE, e));
```

Comment:
Use "info" or "warn" instead of "error" in the log level. Moreover, you could specify a better error message because this is an important step in code logic.